app/presenters/downstream_presenter.rb

```diff
@@ -59,7 +59,7 @@ def access_limit
     end
 
     def location
-      @location ||= Location.find_by!(content_item: content_item)
+      @location ||= Location.find_by(content_item: content_item)
```

Comment:
This may be overkill, but it seems like location without a base path is now part of the domain, so having something like:
`Location.find_by(content_item: content_item) || NoBasePathLocation.new` (bad name, maybe `NilLocation` or `GovernmentLocation` or whatever) and that guy has a method on it called `base_path` which is empty? It avoids that try down there :)